FFMPEG_INSTALLATION.rst

```diff
@@ -0,0 +1,11 @@
+=========
```

Comment:
.rst conventions are that the '===' needs to have the same length as the title text